### ISSUE TITLE
Video only working after 5 seconds in Apple Homekit

### ISSUE THREAD

**Issue by georgeboot** [briis/unifiprotect#267]
I have the camera added with a separate accessory mode homekit instance.

When I open the video stream in Homekit, the camera footage loads after ~ 1 seconds. However, only half of the image:
<img width="1081" alt="Schermafbeelding 2021-07-20 om 16 21 44" src="https://user-images.githubusercontent.com/884482/126341039-a57c75d0-644c-44f2-bd90-e8a445c653f1.png">

Only after about 3-5 seconds, the rest of the image loads. After that, it works as it should.

When I load the camera from HomeAssistnat, it works perfectly, no green sections. The initial load time is also ~1 second.

I previously had the camera set-up using the default bridge and with that, the video actually worked perfectly. No audio though.

Audio actually also isn't perfect now. It comes and goes away every 3 seconds or so, but that I can live with for now. Thought this is already tracked in #193

**Comment by briis** [comment]
Sorry. I am not a homekit expert, so I cannot really help with this. Try nd ask in the HA forum, there are the experts on getting things properly in to homekit.

**Comment by bdraco** [comment]
Home Assistant will wait for a key frame before displaying the video.  HomeKit will show it right away.  You'll notice the video is delayed a bit in Home Assistant, but not HomeKit.   There isn't anything we can do to make the green frame go away without delaying the video for up to ~10s

**Comment by bdraco** [comment]
https://community.ui.com/questions/Override-Keyframe-Interval-on-UVC-G4-PRO-UniFi-Protect/1dcfc55d-a374-4c21-9a21-6852602ef2aa

**Comment by bdraco** [comment]
Additional detail here: https://github.com/briis/unifiprotect/issues/199#issuecomment-834461773

**Comment by briis** [comment]
No Activity for 3 weeks, so I will close this now. Please reopen if you have further questions.
